sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/meta/provider/pubsub/PubsubIOJsonTable.java

```diff
@@ -122,14 +115,14 @@
 @Experimental
 class PubsubIOJsonTable extends BaseBeamTable implements Serializable {
```

Comment:
This class doesn't actually have anything specific to pubsub in it anymore, it's just a generic wrapper that could be used for any `SchemaIO`. Could you rename it to something like `SchemaIOTable` and remove references to pubsub?

The javadoc for this class should also be removed and incorporate into the one for `PubsubSchemaCapableIOProvider` (but without the specific examples using SQL syntax).

Reply:
Making the provider and table classes generic is being worked on in another branch. The renaming corrections will be part of a separate PR.